src/CmSign.php

```diff
@@ -534,6 +533,10 @@ public function parseDocumentFieldsFile(string $path, string $documentId)
                 );
             }
             $row = new Field($fieldType, $documentId, $fieldLocations);
+            if(is_array($group) && count($group) > 0 && isset($group[0]["niet verplicht"]) && (int)$group[0]["niet verplicht"] == 1) :
```

Comment:
The `$group` can only be an array and will always have at least one field. After the changes on line 507 you can also remove the `isset` check. Which leaves the value check.